packages/components/src/index.js

```diff
@@ -165,3 +164,5 @@ export {
 	__unstableWithNext,
 	__unstableComponentSystemProvider,
 } from './__next/context';
+
+export { useInlineEdit } from './inline-edit';
```

Comment:
This should be marked unstable at first.

```suggestion
export { useInlineEdit as __unstableUseInlineEdit } from './inline-edit';
```